airflow/contrib/hooks/gcp_api_base_hook.py

```diff
@@ -48,27 +61,27 @@ def _authorize(self):
         Returns an authorized HTTP object to be used to build a Google cloud
         service hook connection.
         """
-        service_account = self._get_field('service_account', False)
         key_path = self._get_field('key_path', False)
         scope = self._get_field('scope', False)
 
         kwargs = {}
         if self.delegate_to:
             kwargs['sub'] = self.delegate_to
 
-        if not key_path or not service_account:
-            logging.info('Getting connection using `gcloud auth` user, since no service_account/key_path are defined for hook.')
+        if not key_path:
+            logging.info('Getting connection using `gcloud auth` user, since no key file is defined for hook.')
             credentials = GoogleCredentials.get_application_default()
-        elif self.scope:
-            with open(key_path, 'rb') as key_file:
-                key = key_file.read()
-                credentials = SignedJwtAssertionCredentials(
-                    service_account,
-                    key,
-                    scope=self.scope,
-                    **kwargs)
         else:
-            raise AirflowException('Scope undefined, or either key_path/service_account config was missing.')
+            if not scope:
+                raise AirflowException('Scope should be defined when using a key file.')
+            scopes = scope.split(',')
```

Comment:
Wonder if we should:

    [trim(s) for s in scope]

To handle extra white space (e.g. `http://foo, http:/bar`)